GVFS/GVFS.UnitTests/Virtualization/Projection/LazyUTF8StringTests.cs

```diff
@@ -299,6 +299,14 @@ public unsafe void NonASCIICharacters_Compare()
                 });
         }
 
+        [TestCase]
+        public void MinimumPoolSize()
+        {
+            LazyUTF8String.ResetPool(new MockTracer(), 0);
+            LazyUTF8String.FreePool();
+            LazyUTF8String.InitializePools(new MockTracer(), 0);

```

Comment:
Can we also validate that `LazyUTF8String.Length` is 100 after `ResetPool` and `InitializePools`?

Reply:
I thought about validating, but the number 100 is not special (and in the new implementation is hidden).

Reply:
You could validate that it is greater than 0

Reply:
I do think we can add a verification that the value is strictly larger than 0. Which is the important thing, and gives a clear indicator of "what are we testing here?" (instead of just "don't throw").

Reply:
(I'm not used to GitHub not updating automatically while the browser is open.)